schema/pgsql/schema.sql

```diff
@@ -129,6 +129,14 @@ CREATE TABLE source (
     -- will likely need a distinguishing value for multiple sources of the same type in the future, like for example
     -- the Icinga DB environment ID for Icinga 2 sources
 
+    -- listener_password_hash is required to limit API access for incoming connections to the Listener. The username is
+    -- "source-${id}", allowing an early verification before having to parse the POSTed event.
+    -- The hash is PHP's password_hash with the PASSWORD_DEFAULT algorithm, as already being used in Icinga Web 2.
+    -- https://icinga.com/docs/icinga-web/latest/doc/20-Advanced-Topics/#manual-user-creation-for-database-authentication-backend
```

Comment:
Wouldn't it be more helpful to say that this must be a bcrypt hash starting with `$2y$`? I mean not sure how often PHP changes `PASSWORD_DEFAULT`, but when the web module writes to this column, it would be a good idea to explicitly specify bcrypt when hashing the value.

Reply:
I like this idea. Now it is double checked during DB insertion as well as during comparison by `golang.org/x/crypto/bcrypt`.